src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2586,6 +2586,104 @@ public function get_test_persistent_object_cache() {
 		return $result;
 	}
 
+	/**
+	 * Calculate total amount of autoloaded data.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @return int autoloaded data in bytes.
+	 */
+	public function get_autoloaded_options_size(): int {
+		$alloptions = wp_load_alloptions();
+
+		$total_length = 0;
+
+		foreach ( $alloptions as $option_name => $option_value ) {
+			$total_length += strlen( $option_value );
+		}
+
+		return $total_length;
+	}
+
+	/**
+	 * Callback for autoloaded_options test.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @return array The test results.
```

Comment:
Arrays in return values can be documented using the same @type annotations described in [the docblock section of the handbook](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#docblock-formatting:~:text=1.1%20Parameters%20That%20Are%20Arrays)

Reply:
@joemcgill, all other test functions have similar documentation. Do you want us to update the return documentation for this function?

Reply:
That's a good point. I think consistency is fine 👍🏻